src/components/Route/CreateRoute.tsx

```diff
@@ -0,0 +1,195 @@
+import { Button, FormControl, Input, Text, VStack, Select, HStack } from 'native-base';
+import { useState } from 'react';
+import { createRoute } from '../../xplat/api';
+import Popup from 'reactjs-popup';
+import 'reactjs-popup/dist/index.css';
+import { User } from '../../xplat/types/user';
+import { RouteType, RouteClassifier, RouteColor, RouteTech } from '../../xplat/types/route';
```

Comment:
wait, RouteColor enum does not exist in this branch's version of xplat. Or perhaps im mistaken? shouldnt this be included in another commit where u merge new xplat?

Reply:
I didn't update xplat before adding in RouteColor. but I just updated it in the latest commit so this is no longer inconsistent.